### PR TITLE
Refactor error handling in /upload/:tenant route

### PR DESCRIPTION
**Pull request by uxjp**
Motivated by this sujestion https://github.com/GeoSales-Evolution/multi-tenant-uploader/pull/25#discussion_r1192466354

### CODE REVIEW

File: api/api.ts [hunk 1]
```diff
@@ -8,54 +8,57 @@ dotenvConfig()
...
-    } else {
-        res.status(400).end('Bad Request')
+        })
+    } catch (error: any) {
+        console.error(error)
+        if (error.message === 'Missing Authorization header' || error.message === 'Missing bearer token') {
+            res.status(401).end(error.message)
+        } else if (error.message === 'Bad Request') {
+            res.status(400).send(error.message)
+        } else {
```
Comment (brenoaraujolima) [inline api/api.ts:59]: Missed the case: `Error fetching Authentication API` ?

File: api/api.ts [hunk 2]
```diff
@@ -8,54 +8,57 @@ dotenvConfig()
...
-                console.error(authError)
-                res.status(500).send('Error fetching Authentication API')
-            })
-    } else {
-        res.status(400).end('Bad Request')
+        })
+    } catch (error: any) {
+        console.error(error)
+        if (error.message === 'Missing Authorization header' || error.message === 'Missing bearer token') {
+            res.status(401).end(error.message)
```
Comment (brenoaraujolima) [inline api/api.ts:56]: Why this one is `end` and the others are `send` ?

File: api/api.ts (hunk 1 above)
Comment (uxjp) [reply api/api.ts:59]: No, it goes in the else and is thrown in line 34

Comment (brenoaraujolima) [reply api/api.ts:59]: You're right. Resolving conversation

**Pull request merged**